framework/SupervisedLearning/ARMA.py

```diff
@@ -1102,6 +1152,120 @@ def finalizeGlobalRomSegmentEvaluation(self, settings, evaluation, weights=None)
         evaluation[target] = self._transformThroughInputCDF(evaluation[target], dist, weights)
     return evaluation
 
+  ### Peak Picker ###
+  def _peakPicker(self,signal,low):
+    """
+      Peak picker
```

Comment:
Please add a little more description to help a casual developer understand what these methods mean, just for ease of use.